angular/src/scss/bwicons/styles/style.scss

```diff
@@ -84,12 +84,12 @@ $icomoon-font-path: "~@bitwarden/jslib-angular/src/scss/bwicons/fonts/" !default
 
 .bwi-li {
   position: absolute;
-  left: -(30em / 14);
-  width: (30em / 14);
-  top: (2em / 14);
+  left: calc(30em / 14) * -1;
```

Comment:
Wouldn't the `* -1` go inside of the `calc()` function? Order or operations for deriving a negative result shouldn't matter in these cases anyway.

Reply:
Do we need the `* -1` trick or does calc support that natively using `calc(-30em/14)`?

Reply:
After doing some digging it appears that as long as we supply a unit of measurement the negative value will be calculated properly. I will update the PR to remove  `* -1` trick